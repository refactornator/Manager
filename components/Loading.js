import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export default () => (
  <View style={styles.container}>
    <Text>Loading...</Text>
  </View>
);

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    justifyContent: 'center',
    alignItems: 'center'
  }
});
