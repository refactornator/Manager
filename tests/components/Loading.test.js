import React from 'react';
import renderer from 'react-test-renderer';

import Loading from '../../components/Loading';

it('renders a simple loading message', () => {
  const rendered = renderer.create(<Loading />).toJSON();
  expect(rendered).toMatchSnapshot();
});
