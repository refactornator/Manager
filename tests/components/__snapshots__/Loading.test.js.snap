// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a simple loading message 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Loading...
  </Text>
</View>
`;
